src/sagas/route.js

```diff
@@ -133,8 +133,10 @@ export function* routeUpdated(
       const { redirectURL } = JSON.parse(atob(payload));
       // Wait until we're actually authenticated
       yield take(SESSION_AUTHENTICATED);
-      // Redirect to the initially requested URL
-      yield put(updatePath(redirectURL));
+      // Redirect to the initially requested URL (if any)
+      if (redirectURL) {
```

Comment:
If we do not proceed with a redirect (at all), this might mean we'll keep the auth token in the URL hash. How about falling back to `"/"` as a default redirect?